install_requirements.py

```diff
@@ -78,6 +80,16 @@ def python_is_compatible():
         else:
             print(f"Error: {arg} must follow --pybind")
             sys.exit(1)
+    elif arg == "--clean":
+        print("Cleaning build artifacts...")
+        print("Cleaning pip-out/...")
+        shutil.rmtree("pip-out/", ignore_errors=True)
+        dirs = glob.glob("cmake-out*/")
```

Comment:
Looks like the mediatek scripts create a directory called `cmake-android-out`. We should standardize them more. Or even put everything under a top-level `out` directory so we could just delete that.